packages/@okta/vuepress-theme-prose/components/Footer.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <footer>
     <p>
-      <span class="italic">Need help?</span> Ask on the
+      <span class="italic">Need support?</span> Contact
+      <SmartLink
+        :item="{ link: 'mailto:developers@okta.com' }"
+        classes="link link--underline"
+      >
+        developers@okta.com
+      </SmartLink>
```

Comment:
Perhaps this is due to IDE autoformatting but the closing tag for this link has to be pressed up against the end of the link or else it will cause the space after the link to be underlined as well